packages/flutter/lib/src/foundation/platform.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The platform that user interaction should adapt to target.
+enum TargetPlatform {
```

Comment:
Do we want suffixes here? Or maybe we're just assuming that the adaptation matches whatever the current version of the platform was when Flutter was built?